test/lock.cc

```diff
@@ -215,6 +238,8 @@ TEST(LockTest, SynchronizableWait) {
 
   Foo foo;
 
+  // TODO(benh): document why we std::move here, rather than just having 1
+  // variable.
```

Comment:
Looks like we're specifically testing that `std::move()` is a valid thing to do (i.e., you can still call the operation and everything works).

Reply:
Deleted.